test/e2e/lib/components/sidebar-component.js

```diff
@@ -86,12 +86,12 @@ export default class SidebarComponent extends AsyncBaseContainer {
 	}
 
 	async selectPlans() {
-		await this.expandDrawerItem( 'Upgrades' );
+		await this.expandDrawerItem( 'Upgrades\nPremium' );
```

Comment:
It seems that `Premium` is generally hardcoded 
https://github.com/Automattic/wp-calypso/blob/de7f1aad7312fd085b49a398b887692b0be8b6eb/test/e2e/specs/wp-plan-purchase-spec.js#L55